site/pages/api/pledge/index.ts

```diff
@@ -13,10 +42,22 @@ export default async function handler(
         if (!signature) {
           return res.status(400).json({ message: "Bad request" });
         }
-
+        await pledgeSchema.validate(req.body.pledge);
```

Comment:
When thsi throws, what does the sent errors object look like client side?

Reply:
It will be the default error I think. In theory it will only throw if someone is trying to use the api without our front end bc the form values are validated before they reach this point.

Reply:
Agreed!